rosidl_typesupport_introspection_cpp/include/rosidl_typesupport_introspection_cpp/field_types.hpp

```diff
@@ -21,30 +21,53 @@
 namespace rosidl_typesupport_introspection_cpp
 {
 
+/// Primitive (plain-old data) types supported by the ROS type system.
```

Comment:
I'm still not sure if these are all POD or not.

Reply:
removed and added the link to the design document 09633719d457cad2e5df479b37d1788907aa2bbc